dali/python/bundle-wheel.sh

```diff
@@ -88,7 +88,6 @@ make_wheel_record() {
 }
 
 DEPS_LIST=(
-    "lib/libdali.so"
```

Comment:
If we don't run libdali.so through the patchelf steps then it means there can be only one libdali in the process space at a time -- are we ok with that?

Reply:
@cliffwoolley - when I'm think about it do we know about any use case when we want to have multiple instances in one process space at the time?

Reply:
@JanuszL -- today probably not, but what if later on some other plugin is using DALI's (eventual) C++ interface and links in libdali.so.someotherversion -- wouldn't we want to defend against loader conflicts for that?

Reply:
@cliffwoolley - fair point. Do we know how TF deals with that?

Reply:
As I discussed with @cliffwoolley we will add a warning in the documentation about the limitation of having only one version of libdali.so loaded by the process and think about a better solution later on.